### PR TITLE
Update TF and tflint versions

### PR DESCRIPTION
**Pull request by kddejong**
## Proposed changes

- Update Terraform and TFLint Versions

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (changes to code, which do not change application behavior)

## Checklist

- [ ] I have filled out this PR template
- [ ] I have read the [CONTRIBUTING](https://github.com/Optum/dce/blob/master/docs/CONTRIBUTING.md) doc
- [ ] I have added automated tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (`README.md`, inline comments, etc.)
- [ ] I have updated the `CHANGELOG.md` under a `## next` release, with a short summary of my changes

### CODE REVIEW

File: modules/alarms_sns.tf [hunk 1]
```diff
@@ -25,7 +25,7 @@ resource "aws_sns_topic_policy" "alarms_policy" {
           "SNS:DeleteTopic",
           "SNS:AddPermission"
         ],
-        "Resource": "aws_sns_topic.alarms_topic.arn",
+        "Resource": "${aws_sns_topic.alarms_topic.arn}",
```
Comment (nathanagood) [inline modules/alarms_sns.tf]: It looks like everywhere else you've _removed_ the `${..}` notation... why adding them here?

Comment (kddejong) [reply modules/alarms_sns.tf]: Its because inside a text blob. ```policy = <<POLICY```.  I believe this is actually just a fix.

Comment (nathanagood) [reply modules/alarms_sns.tf]: Awesome. I'll approve, LGTM

**Comment by kddejong** [comment]
@eschwartz the terraform plan from master to this version is clean.  The fix will be detected for the SNS policy but other than that there are no other changes.

File: pipelines/pipeline.yml [hunk 2]
```diff
@@ -44,9 +44,15 @@ stages:
             displayName: "Set up Golang CI Tools"
 
           # Unit Tests
-          - script: |
-              make test
+          - task: AWSShellScript@1
             displayName: "Unit Tests"
+            inputs:
+              awsCredentials: "AWS_Login"
```
Comment (eschwartz) [inline pipelines/pipeline.yml]: @kddejong what's the story here?
I think we actually need to revert this -- I'll message you in Flowdock with details

Comment (eschwartz) [reply pipelines/pipeline.yml]: band-aid fix put it. We'll have to revisit this, if we start getting some community PRs

**Pull request merged**